spec/factories/custom_field_masters.rb

```diff
@@ -5,6 +5,12 @@
 FactoryBot.define do
   factory :custom_field_master do
     nonprofit { '' }
-    name { 'MyString' }
+    name { 'Def Name' }
+    factory :custom_field_definition_with_nonprofit do 
+      name { 'Def Name' }
+      nonprofit { create(:fv_poverty)}
+    end
   end
+
+  
```

Comment:
I think this file could be removed from rubocop ignored files!

Reply:
Done!